libraries/ESP8266WebServer/src/Parsing-impl.h

```diff
@@ -263,14 +263,14 @@ bool ESP8266WebServerTemplate<ServerType>::_parseRequest(ClientType& client) {
 }
 
 template <typename ServerType>
-bool ESP8266WebServerTemplate<ServerType>::_collectHeader(const char* headerName, const char* headerValue) {
-  for (int i = 0; i < _headerKeysCount; i++) {
-    if (_currentHeaders[i].key.equalsIgnoreCase(headerName)) {
-            _currentHeaders[i].value=headerValue;
+bool ESP8266WebServerTemplate<ServerType>::_collectHeader(const String& headerName, const String& headerValue) {
```

Comment:
Signature change

Reply:
reverted, this was actually for another PR